keras_cv/layers/object_detection/target_gather.py

```diff
@@ -12,16 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Optional
-
 import tensorflow as tf
 
 
 def _target_gather(
```

Comment:
I don't think this belongs in `layers` -- maybe `utils`?

Reply:
keras_cv/utils/ for now?  I had it in here because  it is only used by OD layers - perhaps a utils file in `keras_cv/layers/object_detection/utils.py`?

Reply:
We already have keras_cv/utils/...
Maybe we should put it there?

Reply:
moved - done